exercises/practice/parallel-letter-frequency/.docs/instructions.append.md

```diff
@@ -0,0 +1 @@
+As a stretch goal, consider if your implementation
```

Comment:
```suggestion
# Instructions append 

As a stretch goal, consider if your implementation
```

I also think that you're missing some words at the end.

Reply:
yep, should be all good now, but do tell if you think it's not suitable